lib/jnpr/junos/utils/ftp.py

```diff
@@ -68,7 +68,12 @@ def put(self, local_file, remote_path=None):
                     remote_file = os.path.join('/tmp/', local_file)
                 else:
                     remote_file = os.path.join(remote_path, local_file)
-            self.storbinary('STOR ' + remote_file, open(local_file, 'rb'))
+
+            if 'callback' in self._ftpargs:
+                self.storbinary('STOR ' + remote_file, open(local_file, 'rb'),
```

Comment:
Hi @ipmonk

Can you change the call to 
self.storbinary(cmd='STOR ' + remote_file, fp=open(local_file, 'rb'), callback=self._ftpargs['callback'])

We don't need to pass blocksize from here.